forge/src/lib.rs

```diff
@@ -79,6 +79,8 @@ pub mod test_helpers {
             gas_limit: 18446744073709551615,
             chain_id: Some(foundry_common::DEV_CHAIN_ID),
             tx_origin: Address::from_str("00a329c0648769a73afac7f9381e08fb43dbea72").unwrap(),
+            block_number: 1,
+            block_timestamp: 1,
```

Comment:
Also needs to be changed here: https://github.com/foundry-rs/foundry/blob/a97090d6eb087184f2a85965f3e15206b8ba97bc/evm/src/executor/opts.rs#L11

This is just for the tests internally

Reply:
good catch